presto-hive/src/main/java/com/facebook/presto/hive/metastore/ThriftHiveMetastore.java

```diff
@@ -542,6 +542,16 @@ public void dropPartition(String databaseName, String tableName, List<String> pa
     public void grantTablePrivileges(String databaseName, String tableName, String grantee, Set<PrivilegeGrantInfo> privilegeGrantInfoSet)
     {
         try {
+            Set<HivePrivilegeInfo> existingPrivileges = getTablePrivileges(grantee, databaseName, tableName);
+
+            Set<PrivilegeGrantInfo> privilegesToGrant = privilegeGrantInfoSet.stream()
```

Comment:
Actually I'm not too convinced about this rename. You decide, whatever you prefer.